src/m365/spo/commands/hubsite/hubsite-get.ts

```diff
@@ -83,14 +86,22 @@ class SpoHubSiteGetCommand extends SpoCommand {
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     try {
+      if (args.options.includeAssociatedSites) {
+        await this.warn(logger, `Parameter 'includeAssociatedSites' is deprecated. Please use 'withAssociatedSites' instead`);
+      }
+
       const spoUrl = await spo.getSpoUrl(logger, this.debug);
       const hubSite = args.options.id ? await this.getHubSiteById(spoUrl, args.options) : await this.getHubSite(spoUrl, args.options);
 
       if (args.options.includeAssociatedSites && (args.options.output && args.options.output !== 'json')) {
         throw 'includeAssociatedSites option is only allowed with json output mode';
       }
 
-      if (args.options.includeAssociatedSites === true && args.options.output && !cli.shouldTrimOutput(args.options.output)) {
+      if (args.options.withAssociatedSites && (args.options.output && args.options.output !== 'json')) {
+        throw 'withAssociatedSites option is only allowed with json output mode';
+      }
```

Comment:
could we combine this condition with the above one? Doing something like 
```
if ((args.options.includeAssociatedSites || args.options.withAssociatedSites) && (args.options.output && args.options.output !== 'json'))
```

and then in the `throw` check which option was set (so is not empty) and then based on that show either `includeAssociatedSites` or `withAssociatedSites`